controllers/wavefrontalert_controller.go

```diff
@@ -105,39 +133,123 @@ func (r *WavefrontAlertReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 		if exist && &wfAlert.Status.ExportParamsChecksum != nil {
 			if reqChecksum != wfAlert.Status.ExportParamsChecksum {
```

Comment:
So, think about it even if you want to apply the changes for non Golang template you can't do that since Golang template is going to fail when you try to excuse the template with lets say 4(3 + new field)  fields with what we have 3 in AlertsConfig. So, It is users decision if they want to apply to all the alerts then they can not combine it with exportParam change